concordium-contracts-common/src/schema_json.rs

```diff
@@ -964,6 +999,109 @@ impl Type {
+            Self::I8 => serde_json::Value::String("<Int8>".to_string()),
+            Self::I16 => serde_json::Value::String("<Int16>".to_string()),
+            Self::I32 => serde_json::Value::String("<Int32>".to_string()),
+            Self::I64 => serde_json::Value::String("<Int64>".to_string()),
+            Self::I128 => serde_json::Value::String("<Int128>".to_string()),
+            Self::ILeb128(_) => {
+                serde_json::Value::String("<String with signed integer>".to_string())
+            }
+            Self::ULeb128(_) => {
+                serde_json::Value::String("<String with unsigned integer>".to_string())
+            }
+            Self::Amount => serde_json::Value::String("<Amount>".to_string()),
+            Self::AccountAddress => serde_json::Value::String("<AccountAddress>".to_string()),
+            Self::ContractAddress => {
+                let mut contract_address = Map::new();
+                contract_address.insert("index".to_string(), Type::U64.schema_type_to_json());
+                contract_address.insert("subindex".to_string(), Type::U64.schema_type_to_json());
+                serde_json::Value::Object(contract_address)
+            }
+            Self::Timestamp => serde_json::Value::String("<Timestamp>".to_string()),
+            Self::Duration => serde_json::Value::String("<Duration>".to_string()),
+            Self::Pair(a, b) => {
+                serde_json::Value::Array(vec![a.schema_type_to_json(), b.schema_type_to_json()])
+            }
+            Self::List(_, element) => serde_json::Value::Array(vec![element.schema_type_to_json()]),
+            Self::Array(_, element) => {
+                serde_json::Value::Array(vec![element.schema_type_to_json()])
+            }
+            Self::Set(_size, element) => {
+                serde_json::Value::Array(vec![element.schema_type_to_json()])
+            }
+            Self::Map(_size, key, value) => {
+                serde_json::Value::Array(vec![serde_json::Value::Array(vec![
+                    key.schema_type_to_json(),
+                    value.schema_type_to_json(),
+                ])])
+            }
+            Self::ContractName(_) => {
+                let mut contract_name = Map::new();
+                contract_name.insert(
+                    "contract".to_string(),
+                    serde_json::Value::String("<String>".to_string()),
+                );
+                serde_json::Value::Object(contract_name)
+            }
+            Self::ReceiveName(_) => {
+                let mut receive_name = Map::new();
+                receive_name.insert(
+                    "contract".to_string(),
+                    serde_json::Value::String("<String>".to_string()),
+                );
+                receive_name
+                    .insert("func".to_string(), serde_json::Value::String("<String>".to_string()));
+                serde_json::Value::Object(receive_name)
```

Comment:
Have you considered using the [`json!`](https://docs.rs/serde_json/latest/serde_json/macro.json.html) macro?
I think it could improve the readability of the code quite a bit.